### PR TITLE
GH-2188 - Adjustments for Ghostery Browser

### PR DESCRIPTION
**Pull request by christophertino**
- Disable intro hub
- Disable Rewards
- Enable metrics
 See GH-2188

### CODE REVIEW

File: app/panel/components/Help.jsx [hunk 1]
```diff
@@ -23,9 +26,11 @@ const Help = () => (
 		<div className="row">
 			<div className="small-12 columns">
 				<h1>{t('panel_help_panel_header')}</h1>
-				<div className="support-section">
-					<a href="#" onClick={openHubPage}>{t('panel_help_setup')}</a>
-				</div>
+				{ BROWSER_INFO.name !== 'ghostery_desktop' && (
```
Comment (sammacbeth) [inline app/panel/components/Help.jsx:29]: So we completely disable access to the hub page on the browser? Would it not be useful for some users who may want to access it later?

File: src/background.js [hunk 2]
```diff
@@ -1642,9 +1648,9 @@ function initializeGhosteryModules() {
 				conf.enable_ad_block = !adblocker.isDisabled;
 				conf.enable_anti_tracking = !antitracking.isDisabled;
 				conf.enable_human_web = !humanweb.isDisabled;
-				conf.enable_offers = !offers.isDisabled && !IS_ANDROID;
+				conf.enable_offers = !offers.isDisabled && !IS_ANDROID && BROWSER_INFO.name !== 'ghostery_desktop';
```
Comment (sammacbeth) [inline src/background.js:1651]: Is there a potential race here? How early is this block evaluated in the startup? The browser name will not be correct until after `chrome.runtime.getBrowserInfo` has resolved.

File: src/classes/ConfData.js [hunk 3]
```diff
@@ -111,8 +111,8 @@ class ConfData {
 			_initProperty('enable_click2play', true);
 			_initProperty('enable_click2play_social', true);
 			_initProperty('enable_human_web', !IS_CLIQZ && !IS_FIREFOX);
-			_initProperty('enable_metrics', false);
-			_initProperty('enable_offers', !IS_CLIQZ && !IS_FIREFOX && !IS_ANDROID);
+			_initProperty('enable_metrics', BROWSER_INFO.name === 'ghostery_desktop');
```
Comment (sammacbeth) [inline src/classes/ConfData.js:114]: Again, could this be a race?

File: app/panel/components/Help.jsx (hunk 1 above)
Comment (christophertino) [reply app/panel/components/Help.jsx:29]: We would need to adjust opt-in default settings for the hub but it's doable.

**Review by ang557 — COMMENTED** [review]
I am being hacked this has been going on for 4 months now there hacking me off your website with coding

File: src/background.js (hunk 2 above)
Comment (christophertino) [reply src/background.js:1651]: This evaluates pretty far down the init chain and is nested inside `cliqz.start()`. The line directly below uses a similar check for android and has been working ok. But I admit I don't love this implementation. Open to suggestions.

File: src/classes/ConfData.js (hunk 3 above)
Comment (christophertino) [reply src/classes/ConfData.js:114]: Same as above, it does evaluate in order but the possibility for a race condition technically exists.

File: src/background.js (hunk 2 above)
Comment (sammacbeth) [reply src/background.js:1651]: We could attach a `Globals.browserInfoReady` Promise in `Globals` when we call `buildBrowserInfo`, which we can then just `await` at these points that need to be sure that the browser check has been done.

File: app/panel/components/Help.jsx (hunk 1 above)
Comment (christophertino) [reply app/panel/components/Help.jsx]: Fixed

File: manifest.json [hunk 4]
```diff
@@ -3,10 +3,11 @@
 	"author": "Ghostery",
 	"name": "__MSG_name__",
 	"short_name": "Ghostery",
-	"version": "8.5.3",
-	"version_name": "8.5.3",
+	"version": "8.5.4",
+	"version_name": "8.5.4",
 	"default_locale": "en",
 	"description": "__MSG_short_description__",
+	"debug": true,
```
Comment (sammacbeth) [inline manifest.json:10]: Should this be here?

Comment (christophertino) [reply manifest.json:10]: yea it's fine for `develop`. It gets removed by the builder.

**Review by sammacbeth — APPROVED** [review]

**Pull request merged**